wcurl

```diff
@@ -49,25 +49,30 @@ usage()
     cat << _EOF_
 ${PROGRAM_NAME} -- a simple wrapper around curl to easily download files.
 
-Usage: ${PROGRAM_NAME} [--curl-options <CURL_OPTIONS>] [--dry-run] [--] <URL>...
-       ${PROGRAM_NAME} [--curl-options=<CURL_OPTIONS>] [--dry-run] [--] <URL>...
+Usage: ${PROGRAM_NAME} [--curl-options <CURL_OPTIONS>]... [-o|-O|--output <PATH>] [--dry-run] [--] <URL>...
+       ${PROGRAM_NAME} [--curl-options=<CURL_OPTIONS>]... [--output=<PATH>] [--dry-run] [--] <URL>...
        ${PROGRAM_NAME} -h|--help
        ${PROGRAM_NAME} -V|--version
 
 Options:
 
-  --curl-options <CURL_OPTIONS>: Specify extra options to be
-                                 passed when invoking curl. May be
+  --curl-options <CURL_OPTIONS>: Specify extra options to be passed when invoking curl. May be
                                  specified more than once.
 
-  --dry-run: Don't actually execute curl, just print what would be
-             invoked.
+  -o, -O, --output <PATH>: Use explicit output path instead of curl's --remote-name logic. If
```

Comment:
We need to document what happens if this value is passed multiple times, I believe that today only the latest value is used.

Implementing logic to allow it to be set multiple times is possible, but we need to decide if it makes sense and check if it will not increase the complexity too much.

Reply:
TODO: Consider making the new parameter behave just like curl's with regards to ordering, from the [manpage](https://manpages.debian.org/unstable/curl/curl.1.en.html#o,):

> You may use this option as many times as the number of URLs you have. For example, if you specify two URLs on > the same command line, you can use it like this:
> 
> curl -o aa example.com -o bb example.net
> 
> and the order of the -o options and the URLs does not matter, just that the first -o is for the first URL and so on, so > the above command line can also be written as
> 
> curl example.com example.net -o aa -o bb
> 
> ...
> 
>  --output is associated with a single URL. Use it once per URL when you use several URLs in a command line.

@sergiodj is going to check how we could do this.